samples/esb/esb_ptx/src/main.c

```diff
@@ -97,7 +98,44 @@ int clocks_start(void)
 	LOG_DBG("HF clock started");
 	return 0;
 }
-#endif /* defined(CONFIG_CLOCK_CONTROL_NRF) */
+
+#elif defined(CONFIG_CLOCK_CONTROL_NRF2)
+
+int clocks_start(void)
+{
+	int err;
+	int res;
+	const struct device *radio_clk_dev =
+		DEVICE_DT_GET_OR_NULL(DT_CLOCKS_CTLR(DT_NODELABEL(radio)));
+	struct onoff_client radio_cli;
+
+	/** Keep radio domain powered all the time to reduce latency. */
+	nrf_lrcconf_poweron_force_set(NRF_LRCCONF010, NRF_LRCCONF_POWER_DOMAIN_1, true);
```

Comment:
Is this the target implementation or a temporary solution? I think it would be good to document that we sacrifice current consumption in favor of lower protocol latency on nRF54. The gazell transmitter is supposed to be low-power device by the original protocol idea. The sample is not power optimized, but it should show good practices, and this probably isn't how the target application would look like.

Reply:
OK, I think the current approach is to have the clock enabled all the time, which is fine for the sample. Users can implement their own approach and e.g. request clock before each transmission if the latency is acceptable.